### PR TITLE
docs: revamp local deployment page

### PR DESCRIPTION
**Pull request by RomainGoussault**
Signed-off-by: Romain Goussault <romain.goussault@owkin.com>

### CODE REVIEW

File: docs/source/contributing/local-deployment.rst [hunk 1]
```diff
@@ -257,9 +239,11 @@ If this command fails and you still have pods up, you can use the following comm
 Next steps
 ==========
 
-Now you are ready to go, you are ready to run either the :doc:`/auto_examples/index` or the :doc:`Substrafl examples </substrafl_doc/examples/index>`.
+Now you are ready to go, you can either run the :doc:`/auto_examples/index` or the :doc:`Substrafl examples </substrafl_doc/examples/index>`.
+
+This local deployment is for developing or testing Substra. If you want to have a more production-ready deployment and a more customized set-up, have a look at the :ref:`deployment section <operations/overview:Overview>`.
 
-If you are interested in more deployment options or more customised set-up, you can have a look at :doc:`/operations/overview` or at the documentation included in the repo of substra_, substra-backend_, orchestrator_ or substra-frontend_.
+Documentation on running tests on any of the Substra components is available on the component repositories, see `substra <https://github.com/substra/substra>`_, `substrafl <https://github.com/substra/substrafl>`_, substra-backend_, orchestrator_, substra-frontend_ and `substra-tests <https://github.com/substra/substra-tests>`_ repositories.
```
Comment (ThibaultFy) [inline docs/source/contributing/local-deployment.rst:246]: Add substra tools ?

File: docs/source/contributing/local-deployment.rst [hunk 2]
```diff
@@ -137,21 +125,14 @@ First time configuration
...
-6. Install frontend dependencies
+5. Install frontend dependencies
 
    .. code-block:: bash
 
       cd substra-frontend
       npm install --dev
 
-7. Update Helm charts
+6. Update Helm charts
```
Comment (ThibaultFy) [inline docs/source/contributing/local-deployment.rst:135]: Add the `cd ..` at the end of each block ? I sometimes finds it confusing when there is "almost" all command to run.

File: docs/source/contributing/local-deployment.rst [hunk 3]
```diff
@@ -257,9 +239,11 @@ If this command fails and you still have pods up, you can use the following comm
 Next steps
 ==========
 
-Now you are ready to go, you are ready to run either the :doc:`/auto_examples/index` or the :doc:`Substrafl examples </substrafl_doc/examples/index>`.
+Now you are ready to go, you can either run the :doc:`/auto_examples/index` or the :doc:`Substrafl examples </substrafl_doc/examples/index>`.
```
Comment (ThibaultFy) [inline docs/source/contributing/local-deployment.rst:242]:
```suggestion
Now you are ready to go, you can either run the :doc:`Substra examples </auto_examples/index>` or the :doc:`SubstraFL examples </substrafl_doc/examples/index>`.
```

**Review by ThibaultFy — CHANGES_REQUESTED** [review]
Some comments, thanks for the updates :D

File: docs/source/contributing/local-deployment.rst (hunk 2 above)
Comment (RomainGoussault) [reply docs/source/contributing/local-deployment.rst:135]: hum no strong opinion about the `cd ..`

File: docs/source/contributing/local-deployment.rst (hunk 1 above)
Comment (RomainGoussault) [reply docs/source/contributing/local-deployment.rst:246]: ah yes always forgetting about substra-tool

Comment (RomainGoussault) [reply docs/source/contributing/local-deployment.rst]: done

**Review by ThibaultFy — APPROVED** [review]
LGTM :)

**Pull request merged**